src/_data/team.yml

```diff
@@ -1084,3 +1084,27 @@
     <p>In his free time, Bogdan enjoys reading books and magazines about current affairs, doing long road trips, hiking and traveling internationally.</p>
   email: bogdan.dumitrescu@codurance.com
   office: London
+
+- id: kirsten_osborn
+  name: Kirsten Osborn
+  image: /assets/custom/img/team/kirsten_osborn.jpg
+  role: Agile Delivery Manager
+  fullDescription:
+    <p>Kirsten has over 10 years’ experience leading strategic business change and operational service improvement.</p>
+    <p>This has involved a combination of service redesign, business process re-engineering and technical delivery. Recent projects have included implementing a new Azure/PowerBI data warehouse platform for Manchester Met University, and leading a customer experience transformation programme at Trafford Council.</p>
+    <p>Kirsten is also studying for an MBA as part of a senior leader apprenticeship at Manchester Met.</p>
+    <p>In her spare time she enjoys road cycling, jazz swing dancing and going to tech meet-ups.</p>
+  email: kirsten.osborn@codurance.com
+  office: Manchester
+
+- id: nicole_smith
+  name: Nicole Smith
+  image: /assets/custom/img/team/nicole_smith.jpg
+  role: New Business Manager
```

Comment:
Is it New Business Manager or just Business Manager?

Reply:
<img width="879" alt="Screenshot 2020-09-29 at 14 43 35" src="https://user-images.githubusercontent.com/57367918/94566379-2b36e300-0262-11eb-86da-bf2690245b69.png">

New Business Manager as per Magda 👍